docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 7
+---
+
+# Integrating Eppo with Webflow
+
+
+In this tutorial, you will learn how how run landing page tests in Webflow after minimal  engineering setup. At a high level, you will add a client side version of Eppo’s Javascript SDK as a script in your Webflow workspace that will:
+
+- initialize the Eppo JS client
+- handle experiment assignment logic
+- dynamically update pre-defined content variables such as text and image URLs
+
+## Prerequisites:
+
+- Eppo Account
+- Webflow account
+- Client-side analytics platform like Segment, Amplitude, Adobe, Mixpanel, etc to capture Eppo assignment events and send them to your data warehouse for analysis.
+
+## Setting up Eppo
+
+Set up an feature flag in Eppo — for step by step instructions, follow our [quickstart](/feature-flags/). We will be using JSON flag type to fill in the content on our Webflow landing page. For the sake of this tutorial, make sure to include the following keys for both variations:
+
+```json
+{
+	"title": "some words",
+	"image": "https://myimage.com",
+	"subtitle": "some more words",
+	"cta": "even more words"
```

Comment:
Note I did this just so I could play around with JSON types. I imagine it may be simpler to just use a string value and our app, define these JSON values in their code, and switch off a string key.

This would also combine nicely with a "default" variation.

```js
const controlValues = {
  "title": "These are the droids you're looking for",
  "image": "obi-wan.jpg",
  "subtitle": "We have more droids than you can shake a light saber at",
  "cta": "Browse Droids"
};

const testValues = {
  "title": "Droids that don't suck",
  "image": "battle-droid.jpg",
  "subtitle": "Obedient. Combat ready. Priced to sell.",
  "cta": "Explore Options"
};

const variationData = window.eppo.getInstance().getStringAssignment(subject, '<FEATURE-FLAG-KEY>') === 'test' ? testValues : controlValues;
```

Reply:
I see what you're saying, the thing we want to demonstrate is the ability for business users to have some autonomy to write their own experiments without code. The current example shows how they can update the values of the variation with the copy and images they want to test without hard-coding it.